### DISCUSSION TITLE
Graph of availability rate on several probes

### DISCUSSION THREAD

**Discussion by Lexalator** [centreon-archive/centreon-archived#10697] [Q&A]
Hello,

I need for my centreon server, to have a graph compiling several probes, and thus have the average availability rate of its probes. Despite my research on the net I can not find a way to achieve this. Do you have any advice please? thanks in advance

**Comment by ponchoh** [comment]
Hello @Lexalator,

Have you considered setting them as a group (Configuration  >  Services  >  Service Groups) and then using "Reporting  >  Dashboard  >  Service Groups"?
